pylint/checkers/similar.py

```diff
@@ -178,12 +181,15 @@ def combine_mapreduce_data(self, linesets_collection):
         self.linesets = [line for lineset in linesets_collection for line in lineset]
 
 
-def stripped_lines(lines, ignore_comments, ignore_docstrings, ignore_imports):
+def stripped_lines(
+    lines, ignore_comments, ignore_docstrings, ignore_imports, ignore_signatures
```

Comment:
```suggestion
    lines, ignore_comments: bool, ignore_docstrings: bool, ignore_imports: bool, ignore_signatures: bool
```
Could you add typing to modified functions signature, please ? We're trying to add them now that we dropped python 3.5 :)